_includes/footer.html

```diff
@@ -31,8 +31,9 @@
   <script src="/js/vendor/jquery-1.10.2.min.js"></script>
   <script src="/js/vendor/bootstrap.min.js"></script>
   <script src="/js/vendor/anchor.min.js"></script>
-  <script src="https://cdn.jsdelivr.net/g/lodash@3.10.1,hogan.js@3.0.2,algoliasearch@3.8.1,autocomplete.js@0.12.0(autocomplete.min.js)"></script>
+  <script src="https://cdn.jsdelivr.net/g/lodash@3.10.1,hogan.js@3.0.2,algoliasearch@3.8.1,autocomplete.js@0.12.0(autocomplete.min.js),requirejs@2.1.20"></script>
   <script src="/js/app/search.js"></script>
+  <script src="/js/app/demo.js"></script>
```

Comment:
Yeah I'm still working it out....there are issues with how we now have a bunch of JS that loads here and we have conflicting JS loading earlier.